src/main/java/ch/njol/skript/lang/Section.java

```diff
@@ -82,7 +84,10 @@ public abstract boolean init(Expression<?>[] expressions,
 	 * (although the loaded code may change it), the calling code must deal with this.
 	 */
 	protected void loadCode(SectionNode sectionNode) {
-		ParserInstance parser = getParser();
+		ParserInstance parser = this.getParser();
+		Set<Annotation> annotations = parser.copyAnnotations();
+		parser.forgetAnnotations(); // scope annotations correctly for section headers
```

Comment:
Doesn't this need to be done for the other loadCode methods too?

Reply:
I think this should be handled by ParserInstance Backup, but please double check.